components/Withdraw.tsx

```diff
@@ -129,11 +129,8 @@ const Withdraw: React.FC<{}> = () => {
     await new Promise(async (resolve) => {
       const unsubHeads = await api.rpc.chain.subscribeNewHeads(async () => {
```

Comment:
you won't need to subscribe the new head.. since the evenProof will only be sent once ready